provider/maas/volumes.go

```diff
@@ -362,8 +362,13 @@ func (mi *maas2Instance) volumes(
 			deviceName := idPath[len(devPrefix):]
 			attachment.DeviceName = deviceName
 		} else if strings.HasPrefix(idPath, devDiskByIdPrefix) {
-			hardwareId := idPath[len(devDiskByIdPrefix):]
```

Comment:
Since we're keeping it in the same API version shouldn't we keep hardwareId as it is, even if we're filling WWN?